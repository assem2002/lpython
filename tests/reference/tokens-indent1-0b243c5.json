{
    "basename": "tokens-indent1-0b243c5",
    "cmd": "lpython --no-color --show-tokens {infile} -o {outfile}",
    "infile": "tests/tokens/errors/indent1.py",
    "infile_hash": "e4b85b69276c943f42a1779fce9d11e4afdcf5a037890a3222afa21d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "tokens-indent1-0b243c5.stderr",
    "stderr_hash": "4aa1509a66bac651d22be84453a3532bb771094363034cd45970bf75",
    "returncode": 1
}