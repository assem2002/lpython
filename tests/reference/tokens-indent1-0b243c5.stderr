tokenizer error: Indentation should be of the same type (either tabs or spaces)
 --> tests/tokens/errors/indent1.py:3:1
  |
3 |   pass
  | ^^ 


Note: Please report unclear or confusing messages as bugs at
https://github.com/lcompilers/lpython/issues.
